packages/webdoc-default-template/Gulpfile.babel.js

```diff
@@ -1,17 +1,27 @@
-import * as gulp from "gulp";
-import * as gutil from "gulp-util";
-import * as path from "path";
-import webpack from "webpack-stream";
+import {
+  join as joinPath,
+  resolve as resolvePath,
+} from "path";
+import gulp from "gulp";
+import gutil from "gulp-util";
 import {name} from "./package.json";
+import webpack from "webpack-stream";
+
 
 gulp.task("build", () => {
   gutil.log(`Building ${name} at ${__dirname}`);
 
-  const entryPath = path.join(__dirname, "./src/app/index.js");
-  const configPath = path.join(__dirname, "./webpack.config.js");
-  const distPath = path.join(__dirname, "./static/scripts/");
+  const entryPath = joinPath(__dirname, "./src/app/index.js");
```

Comment:
I don't understand why we aren't using `path` directly?

Reply:
Just a shorthand.